velox/exec/tests/utils/HiveConnectorTestBase.h

```diff
@@ -20,7 +20,6 @@
 #include "velox/connectors/hive/HiveDataSink.h"
 #include "velox/connectors/hive/TableHandle.h"
 #include "velox/dwio/dwrf/common/Config.h"
```

Comment:
Here it's still dependent on dwrf. Please remove it. Do proper refactor if you need.

Reply:
Removed, build was successful so no need to refactor.